web/styles/typeahead.css

```diff
@@ -72,6 +76,17 @@
                 white-space: nowrap;
                 gap: 3px;
             }
+
+            .compose-stream-name {
+                overflow: visible;
+                gap: 0;
+            }
+        }
+
+        & > span {
+            cursor: default;
+            color: var(--color-compose-chevron-arrow);
+            text-decoration: none;
```

Comment:
This should be written using a class, not just `span` as the inner thing; it's imprecise and leads to both bugs and performance issues to write CSS this way.